android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerModule.kt

```diff
@@ -340,6 +340,13 @@ class RNGestureHandlerModule(reactContext: ReactApplicationContext?) :
     handlerTag: Int,
     config: ReadableMap,
   ) {
+
+    if (registry.getHandler(handlerTag) !== null) {
+      throw IllegalStateException(
+        "Handler with tag $handlerTag already exists. Please ensure that no GestureHandler instance is used across multiple GestureDetectors."
```

Comment:
```suggestion
        "Handler with tag $handlerTag already exists. Please ensure that no Gesture instance is used across multiple GestureDetectors."
```

I believe it only applies to the new API so we can simply stick to `Gesture` name here.

Reply:
Done in [0b178e3](https://github.com/software-mansion/react-native-gesture-handler/pull/2694/commits/0b178e350481ae074caccd46657da242d2f48f65)